sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -354,7 +367,7 @@ export type ComplexDataType = "Edm.ComplexType" | "Collection(Edm.ComplexType)";
 
 // @public
 export interface ComplexField {
-    fields: SearchField[];
+    fields?: SearchField[];
```

Comment:
Could you tell me more about this change? is it just a bugfix so users don't ahve to pass an empty array or is there another reason?